app/assets/stylesheets/thing.scss

```diff
@@ -19,7 +19,20 @@ label.error {
   color: $error;
 }
 
-
+.well {
```

Comment:
I'm okay with doing this in-app, but there is argument for this being added upstream to the theme itself

Reply:
Yeah, I wondered about that too. If you'd like, I can submit this as a PR there.

Reply:
I'm a bit waffly on this. I'm not against this as-is, but think I prefer it be added upstream (not to the theme gem but to the theme itself as the theme-gem should pull it in automatically on a new release) along with updated examples to use it. If you would prefer to not do that yet, we can land this as-is and open a ticket to move this feature upstream as maintenance in the future (noting that we should remove this when we do it). I'm honestly good with either approach.

Reply:
Is the theme itself at mitlib-style? I'm realizing that I may not be entirely sure about this setup.

We can continue this discussion on Monday, I think - I may use the intervening time to get a handle on what, exactly, the theme needs to be on-brand. Our panel implementation is different from what I'm envisioning in this work, and I want to consider that a bit more closely.

Reply:
The more I think about this, the more I kind of want to merge this as-is, and have a ticket in maintenance to add this capability to the theme. I've created https://mitlibraries.atlassian.net/browse/ENGX-105 for that follow-on work, and would like to move forward with merging this at the moment.